### DISCUSSION TITLE
Site Search Improvements

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#1252] [Ideas]
Currently, site search lets you filter by everything, games, achievements, users, forums, or comments. 

However, I think we should be able to filter "comments" by achievement comments, user wall comments, game wall comments, ticket comments, etc. 

Additionally, a lot of newer comment sections are not searchable. This includes admin comments on user pages, dev comments on game pages, managehashes.php comments, manageclaims.php comments, etc. I think leaderboard comments, too?

Speaking of comments: https://github.com/RetroAchievements/RAWeb/discussions/1056 :D

**Comment by Jamiras** [comment]
Handled in #1254
